src/ThothSerializer.fs

```diff
@@ -53,7 +53,7 @@ type ThothSerializer (?isCamelCase : bool, ?extra : ExtraCoders, ?skipNullField
 
     static member ReadBodyRaw (ctx: HttpContext) =
         task {
-            use stream = new System.IO.StreamReader(ctx.Request.Body, Utf8EncodingWithoutBom)
+            use stream = new System.IO.StreamReader(ctx.Request.Body, Utf8EncodingWithoutBom, true, DefaultBufferSize, true)
```

Comment:
Hello @MaxDeg , 

I don't know the `SteamReader`API, why is this one having 3 new parameters instead of just one like the others? 
Should we use the 3 parameters or use something like `?leaveOpen = true`?

Reply:
Hello,

Sadly StreamReader only have one constructor that allow to provide leaveOpen option as last parameter. None of those parameters are optional so we need to provide them all.